src/components/footer.js

```diff
@@ -9,10 +9,18 @@ export default function Footer(options = {}) {
   return Component({
     render: function render() {
       let middleContent = '';
-      if (data.img) {
+      if (data.img && !data.url && !data.urlText) {
+        middleContent = `<img src="${data.img}">`;
+      } else if (data.img && data.url && !data.urlText) {
+        middleContent = `<a href="${data.url}"><img src="${data.img}"></a>`;
+      } else if (data.img && !data.url && data.urlText) {
+        middleContent = `<img src="${data.img}">${data.urlText}`;
+      } else if (data.img && data.url && data.urlText) {
+        middleContent = `<img src="${data.img}"><a href="${data.url}">${data.urlText}</a>`;
+      } else if (data.img) {
         middleContent = `<img src="${data.img}">`;
       } else if (data.url && data.urlText) {
-        middleContent = `<a href="${data.url}">${data.urlText}</a>">`;
+        middleContent = `<a href="${data.url}">${data.urlText}</a>`;
       } else if (data.text) {
```

Comment:
How about replacing text with urlText (or better the other way around)?

Reply:
yeah for sure urlText, although I think this was text is from the original code so not sure if it was intended to be supported.

Reply:
However in case you have no url the urlText is a bit misleading

Reply:
jupp looking at the docs, that is true. Maybe update docs with an statement that urlText is depended that the url for a link is set.

Reply:
yes, and that you cannot combine text with other elements. If you want image and text, you must use urlText.

Reply:
Jupp! 

I will referense this conversion in an issue on the api-docs repo.